distogram/__init__.py

```diff
@@ -326,31 +334,81 @@ def stddev(h: Distogram) -> float:
     return math.sqrt(variance(h))
 
 
-def histogram(h: Distogram, bin_count: int = 100) -> List[Tuple[float, float]]:
+def histogram(
+    h: Distogram, 
+    bin_count: int = 100, 
+    data_type: str = "distogram") -> Union[List[Tuple[float, float]],
```

Comment:
Let's change the result to match the numpy format, so please remove this parameter, and always return numpy format.

Reply:
I've made the changes to the numpy histogram format. I still want to convince myself that there is not an off by one error in the binning.